app/notification.html

```diff
@@ -1,54 +1,9 @@
-<!DOCTYPE html>
+<!doctype html>
 <html dir="ltr">
   <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta charset="utf-8" />
-
-    <% if (it.isTest) { %>
-    <title>MetaMask Dialog</title>
-    <% } else { %>
-    <title>MetaMask</title>
-    <% } %>
-
-    <style>
-      #app-content {
-        display: flex;
-        flex-flow: column;
-      }
-
-      #loading__logo {
-        width: 10rem;
-        height: 10rem;
-        align-self: center;
-        margin: 10rem 0 0 0;
-      }
-
-      #loading__spinner {
-        width: 2rem;
-        height: 2rem;
-        align-self: center;
-        margin-top: 1rem;
-      }
-    </style>
-    <link rel="preload" href="/_locales/en/messages.json" as="fetch" crossorigin="anonymous" />
-    <link rel="stylesheet" href="../ui/css/index.scss" />
+    <%~ include("partial-head.html", {title: it.isTest ? "MetaMask Dialog" : null}) %>
   </head>
   <body class="notification">
-    <div id="app-content">
-      <img
-        id="loading__logo"
-        src="./images/logo/metamask-fox.svg"
-        alt=""
-        loading="lazy"
-      />
-      <img
-        id="loading__spinner"
-        src="./images/spinner.gif"
-        alt=""
-        loading="lazy"
-      />
-    </div>
```

Comment:
Not sure why notifications.html had its own loading HTML and CSS rules; they both ended up rendering the same.